packages/gatsby-transformer-cloudinary/api/index.js

```diff
@@ -1,10 +1,12 @@
-const {
-  getFixedImageObject,
-  getFluidImageObject,
-} = require('../gatsby-image/get-image-objects');
+// delete this file then? Asks Cap'n Ola Vea Jr.
 
-// Runtime APIs for fetching remote images on Cloudinary
-module.exports = {
-  getFixedImageObject,
-  getFluidImageObject,
-};
+// const {
```

Comment:
nitpick(blocking): remove code instead of commenting it out

Reply:
😺

Reply:
😺